src/tools/dev/scripts/bv_support/bv_vtk.sh

```diff
@@ -1611,21 +1903,28 @@ function build_vtk
     else
         vopts="${vopts} -DZLIB_LIBRARY_DEBUG:FILEPATH=${ZLIB_LIBRARY}"
     fi
+    if [[ "$DO_VTK94" == "yes" ]] ; then
+        vopts="${vopts} -DZLIB_BUILD_EXAMPLES:BOOL=false"
+        vopts="${vopts} -DVTK_ENABLE_REMOTE_MODULES:BOOL=OFF"
+    fi
 
     CMAKE_BIN="${CMAKE_INSTALL}/cmake"
     cd ${VTK_BUILD_DIR}
 
+    if [[ "$DO_MESAGL" == "yes" ]] ; then
+        export LD_LIBRARY_PATH="${LLVM_LIB_DIR}:$LD_LIBRARY_PATH"
+    elif [[ "$DO_OSMESA" == "yes" && "$DO_VTK94" == "no" ]] ; then
+        export LD_LIBRARY_PATH="${LLVM_LIB_DIR}:$LD_LIBRARY_PATH"
+    fi
```

Comment:
If the result of the conditionals is the same then can we combine these?
```bash
if [[ "$DO_MESAGL" == "yes" ]] || [[ "$DO_OSMESA" == "yes" && "$DO_VTK94" == "no" ]]; then
    export LD_LIBRARY_PATH="${LLVM_LIB_DIR}:$LD_LIBRARY_PATH"
fi
```

Reply:
done